src/lib/components/composites/criteria/PaperDecisionBanner.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+    import { PaperDecision, Project_Paper } from "$lib/model/api/project";
+    import type { User } from "$lib/model/api/user";
+    import { resource } from "$lib/resource.svelte";
+    import { cn } from "$lib/utils/shadcn-helper";
+    import UserAvatar from "../user-avatar/UserAvatar.svelte";
+    import { Skeleton } from "$lib/components/primitives/skeleton";
+    import UserAvatarSkeleton from "../user-avatar/UserAvatarSkeleton.svelte";
+
+    export interface PaperDecisionBannerProps {
+        reviewers: Promise<User[]>;
+        loadingProjectPaper: Promise<Project_Paper>;
+    }
+
+    const { reviewers: loadingReviewers, loadingProjectPaper }: PaperDecisionBannerProps = $props();
+
+    const bannerColor = resource<Project_Paper, string>(loadingProjectPaper, {
+        initialValue: "bg-unreviewed-gray",
+        onSuccess: (paper) => getDecisionColor(paper.decision),
+        onErrorValue: "bg-unreviewed-gray",
+    });
+    const bannerLabel = resource<Project_Paper, string>(loadingProjectPaper, {
+        initialValue: "",
+        onSuccess: (paper) => getDecisionLabel(paper.decision),
+        onErrorValue: "Couldn't load Paper Decision",
```

Comment:
```suggestion
        onErrorValue: "Couldn't load paper decision",
```

This is a general point we could discuss. You always use uppercase letters at the beginning, but I thought in englisch everything is written in lowercase?!

Reply:
Yes, I thought this looks better and because it's a specific term in our application, I wrote it in upper case. It doesn't really bother me, how we write it.

Reply:
But do we really call it "Paper Decision" somewhere? Imo it is better to dont write in upper case or at least dont mix it.

Reply:
Yes, I agree.